CHANGELOG.md

```diff
@@ -1,4 +1,320 @@
+details can be found in the [DNS README](https://github.com/kubernetes/kubernetes/blob/release-1.2/cluster/addons/dns/README.md#a-records-and-hostname-based-on-pod-annotations---a-beta-feature-in-kubernetes-v12). Changes were introduced in PR [#20688](https://github.com/kubernetes/kubernetes/pull/20688).
+  * New SchedulerExtender enables users to implement custom
+out-of-(the-scheduler)-process scheduling predicates and priority functions,
+for example to schedule pods based on resources that are not directly managed
+by Kubernetes. Changes were introduced in PR [#13580](https://github.com/kubernetes/kubernetes/pull/13580). Example configuration and documentation is available [here](https://github.com/kubernetes/kubernetes/blob/master/docs/design/scheduler_extender.md). This is an alpha feature and may not be supported in its current form at beta
+or GA.
+  * New Flex Volume Plugin enables users to use out-of-process volume plugins that
+are installed to “/usr/libexec/kubernetes/kubelet-plugins/volume/exec/” on
+every node, instead of being compiled into the Kubernetes binary. See [example](https://github.com/kubernetes/kubernetes/blob/master/examples/flexvolume/README.md) for details.
+  * vendor volumes into a pod. It expects vendor drivers are installed in the
+volume plugin path on each kubelet node. This is an alpha feature and may
+change in future.
+
+## Action required
+
+  * Docker v1.9.1 is officially recommended. Docker v1.8.3 and Docker v1.10 are
+supported. If you are using an older release of Docker, please upgrade. Known
+issues with Docker 1.9.1 can be found below.
+  * CPU hardcapping will be enabled by default for containers with CPU limit set,
+if supported by the kernel. You should either adjust your CPU limit, or set CPU
+request only, if you want to avoid hardcapping. If the kernel does not support
+CPU Quota, NodeStatus will contain a warning indicating that CPU Limits cannot
+be enforced.
+  * The following applies only if you use the Go language client (<code>/pkg/client/unversioned</code>) to create Job by defining Go variables of type "<code>k8s.io/kubernetes/pkg/apis/extensions".Job</code>).  We think <strong>this is not common</strong>, so if you are not sure what this means, you probably aren't doing this.  If
+you do this, then, at the time you re-vendor the "<code>k8s.io/kubernetes/"</code> code, you will need to set <code>job.Spec.ManualSelector = true</code>, or else set <code>job.Spec.Selector = nil.  </code>Otherwise, the jobs you create may be rejected.<code> </code> See [Specifying your own pod selector](http://kubernetes.io/docs/user-guide/jobs/#specifying-your-own-pod-selector). 
```

Comment:
Yeah, that looks suspicious. Happy for that to be fixed in a followup. We need these notes in ASAP.